deployment/docker/Dockerfile

```diff
@@ -41,12 +44,10 @@ RUN npm run build
 
 
 WORKDIR /home/web/django_project
-# RUN python manage.py collectstatic --noinput
 ADD ./deployment/docker/uwsgi.conf /uwsgi.conf
 ADD ./deployment/docker/run.sh /run.sh
```

Comment:
Do we need `run.sh`, since we are running entrypoint as `uwsgi` we can delete this file